barman/server.py

```diff
@@ -773,8 +782,20 @@ def check_postgres(self, check_strategy):
                     check="no access to backup functions",
                 )
 
+        self._check_streaming_supported(check_strategy, remote_status)
+
+        self._check_wal_level(check_strategy, remote_status)
+
+        if self.config.primary_conninfo is not None:
+            self._check_standby(check_strategy)
+
+    def _check_streaming_supported(self, check_strategy, remote_status, suffix=None):
         if "streaming_supported" in remote_status:
-            check_strategy.init_check("PostgreSQL streaming")
+            check_name = "PostgreSQL streaming"
+            if suffix is not None:
+                check_name = f"{check_name} ({suffix})"
```

Comment:
```suggestion
            check_name = "PostgreSQL streaming" + ("" if suffix is None else f" ({suffix})")
```
Maybe we could have a single attribution here?